src/client/data/notificationMessages.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Probably want to remove this ones for the final merge

Reply:
yes, dont forget about this one